### ISSUE TITLE
Training with argoverse dataset

### ISSUE THREAD

**Issue by wang-h-w** [hehefan/PointRNN#7]
Hi, @hehefan !
Thanks for your nice work!
In the process of training the argo dataset, I have some problems. I downloaded the original dataset, and processed the data according to the format of the test set you gave. The data can be read normally. After the training, I checked the training log and found that the two evaluation indicators did not drop significantly, but fluctuated within a less than ideal range. The result of the evaluation also did not achieve the expected effect in the paper.
My question is: 1) Is there a problem with my data processing? Are you using all the data? 2) Can you provide train.log so that I think I might be able to find the problem better.
This is the field I just started, and your work has inspired me a lot, thank you.

**Comment by hehefan** [comment]
Hi @haowen1999,

Did you crop the point cloud frames with the range [-5m, 5m]?

The following is my processed data:
 https://drive.google.com/file/d/1Zo-YguAOXFwK8xhv3ICSL_wKhDfZasRl/view?usp=sharing

The training log is https://drive.google.com/file/d/1uDud9b8TZVq8cV3-rJ1nXv7QDAFhrkGs/view?usp=sharing

Thank you.

**Comment by wang-h-w** [comment]
Hi. @hehefan 
Thanks for your answer. I now use your data set for training, the parameters of train-argo-nu.py have not changed, but the training results are still unsatisfactory and have not reached the effect of your training.
I use Tensorflow-gpu=1.9.0, CUDA9.0, gcc-6.3.0/5.5.0. When using the make command, some warning appears, but it can run normally. Is it a problem with the gcc version?

**Comment by wang-h-w** [comment]
@hehefan I downgrade the version of gcc and g++ to 5.4.0, but training results are still unsatisfactory. Are all the parameters you used when training the model by default?
Best wishes!

**Comment by hehefan** [comment]
Hi @haowen1999 ,

I am not sure about the problem, but I do not think it is caused by the g++ version. I used the same parameters in the training script. 

Are the two losses descending during training, even not significantly?  Can you run it normally on the moving mnist dataset? 

Thank you.
